JS/Lesson34/gulcin/recipe-app/src/modules/recipes/NewRecipe/index.jsx

```diff
@@ -0,0 +1,179 @@
+                            minLength: {
+                                value: 50,
+                                message: 'Please enter Instructions at least 50 characters long',
+                            },
+                        })}
+                    />
+                </div>
+                {errors.strInstructions && (
+                    <span className='error-message'>{errors.strInstructions.message}</span>
+                )}
+            </fieldset>
+            <fieldset className='fieldset'>
+                <div className='input-area'>
+                    <label htmlFor="">Ingredients and measures</label>
+                    <textarea placeholder='Milk - 500ml, eggs - 2, sugar - 2 tbsp' {...register('strIngredients', {
+                        required: 'Ingredients are required',
+                        minLength: {
+                            value: 10,
+                            message: 'Please enter Instructions at least 50 characters long',
+                        },
+                    })} />
+                </div>
+                {errors.strIngredients && (
+                    <span className='error-message'>{errors.strInstructions.message}</span>
+                )}
+            </fieldset>
+            <fieldset className='fieldset'>
+                <div className='input-area'>
+                    <label htmlFor='strArea'>Area</label>
+                    <select id='strArea' {...register('strArea', { required: 'Please, choose an area' })}>
+                        <option value=''>Select Area</option>
+                        {areaOfRecipe.map((area) => (
+                            <option
+                                key={area}
+                                value={area}
+                            >
+                                {area}
+                            </option>
+                        ))}
+                    </select>
+                </div>
+                {errors.strArea && (
+                    <span className='error-message'>{errors.strArea.message}</span>
+                )}
+            </fieldset>
+            <fieldset className='fieldset'>
+                <div className='input-area'>
+                    <label htmlFor='strMealThumb'>Photo of the recipe</label>
+                    <input type='file' id='strMealThumb' accept="image/png, image/jpeg" {...register('strMealThumb', { required: 'Photo is required' })} />
+                </div>
+                {errors.strMealThumb && (
+                    <span className='error-message'>{errors.strMealThumb.message}</span>
+                )}
+            </fieldset>
+            <fieldset className='fieldset'>
+                <div className='input-area'>
+                <label htmlFor='strTags'>Tags</label>
+                <input type='text' id='strTags' placeholder='Pudding, dessert' {...register('strTags', { required: 'Tags are required.' })} />
```

Comment:
For consistency, keep all placeholder values in lowercase to show that the tags should be in lower case
```suggestion
                <input type='text' id='strTags' placeholder='pudding, dessert' {...register('strTags', { required: 'Tags are required.' })} />
```